composer.json

```diff
@@ -18,7 +18,7 @@
         }
     ],
     "require": {
-        "php": "^7.2 || ^8.0",
+        "php": "^8.0",
```

Comment:
I know 7.x is EOL. But according to the philosophy of Aura this should be a major version.

Reply:
Yes, the review is more on the design of the attributes. If there is no comment on that, I will continue into the direction I have suggested, using the guidelines and philosophy of Aura.